lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1420,6 +1431,20 @@ func (w *WalletKit) fundPsbtInternalWallet(account string,
 		// generating a new change address.
 		changeIndex, err := w.cfg.Wallet.FundPsbt(
 			packet, minConfs, feeSatPerKW, account, keyScope,
+			func(utxo wtxmgr.Credit, blockHeight int32) bool {
```

Comment:
why does this version of `allowUtxo` also take a block height?

Reply:
Good call, Removed